jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -0,0 +1,50 @@
+from traitlets import List
+
+from jdaviz.configs.imviz.helper import link_image_data
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import TemplateMixin
+from jdaviz.utils import load_template
+
+__all__ = ['LinksControl']
+
+
+@tray_registry('imviz-links-control', label="Imviz Links Control")
+class LinksControl(TemplateMixin):
+    template = load_template("links_control.vue", __file__).tag(sync=True)
+
+    # TODO: Use radio button groups?

```

Comment:
Since each entry here only has two choices, I wonder if grouped radio buttons are nicer. Is that even possible in ipyvuetify framework that we use here, @mariobuikhuizen and @kecnry ?

Reply:
Here is what they would look like as a radio group compared to the current dropdowns.  Pros: you can see both choices without having to open the dropdown.  Cons: without additional styling (and I already removed the wrapping v-containers to avoid even more padding) this takes quite a bit more space and now requires scrolling.
<img width="990" alt="select" src="https://user-images.githubusercontent.com/877591/134683226-89f36195-d118-4b96-959f-0730c12e1443.png">

<img width="990" alt="radio" src="https://user-images.githubusercontent.com/877591/134683233-1f0806aa-e6d2-4712-a74d-38a526f28186.png">

Reply:
Allowing them to be side-by-side if the panel is wide enough does help with any space concerns.  We could also use a switch or toggle button for the affine transform option since that will likely always be a boolean (but I'm guessing WCS fallback scheme could expand to more options down the road
<img width="990" alt="radio_row" src="https://user-images.githubusercontent.com/877591/134685920-9c80eee8-efe4-4c8c-89b3-73fc99e72152.png">
?)

Reply:
In my imagination, it was side by side. In the future, if we add more stuff, we could change it back to drop down, I guess.

Though... it is more likely that we will get rid of this altogether when @astrofrog gets around to #834 (using native Glue menu).

So my thoughts is... if next active POs @camipacifici and @PatrickOgle do not have a preference, then @kecnry can just pick his fav.

Reply:
See https://github.com/pllim/jdaviz/pull/3 for the radio/switch implementation.